lib/AST/ASTMangler.cpp

```diff
@@ -390,7 +391,15 @@ void ASTMangler::appendDeclName(const ValueDecl *decl) {
         break;
     }
   } else {
-    appendIdentifier(decl->getName().str());
+    // FIXME: Should a mangled subscript name contain the string "substring"?
```

Comment:
"subscript" :-) I tend to agree, we might as well pick an abbreviation that isn't used. But we can do that later.